mail/common/src/test/java/com/fsck/k9/helper/MailToTest.java

```diff
@@ -187,4 +188,52 @@ public void testCaseInsensitiveParamWrapper_withoutQueryParameters() {
 
         assertThat(Collections.<String>emptyList(), is(result));
     }
-}
+
+    @Test
+    public void testGetInReplyTo_singleMessageId() {
+        Uri uri = Uri.parse("mailto:?in-reply-to=<7C72B202-73F3@somewhere>");
+
+        MailTo mailToHelper = MailTo.parse(uri);
+
+        assertEquals(mailToHelper.getInReplyTo(), "<7C72B202-73F3@somewhere>");
```

Comment:
You mixed up the order of arguments here and in the other tests. It's `assertEquals(expected, actual)`.